app/src/main/res/drawable/ic_upload_photo.xml

```diff
@@ -1,12 +1,22 @@
-<vector android:height="150dp" android:viewportHeight="50"
-    android:viewportWidth="50" android:width="150dp" xmlns:android="http://schemas.android.com/apk/res/android">
-    <path android:fillColor="#ACACAC" android:fillType="evenOdd"
-        android:pathData="M0,0h50v50h-50z"
-        android:strokeColor="#00000000" android:strokeWidth="1"/>
-    <group>
-        <clip-path android:pathData="M37,37l-24,0l0,-24l24,0z M 0,0"/>
-        <path android:fillColor="#FFFFFF" android:fillType="nonZero"
-            android:pathData="M16,17L16,14L18,14L18,17L21,17L21,19L18,19L18,22L16,22L16,19L13,19L13,17L16,17ZM19,23L19,20L22,20L22,17L29,17L30.83,19L34,19C35.1,19 36,19.9 36,21L36,33C36,34.1 35.1,35 34,35L18,35C16.9,35 16,34.1 16,33L16,23L19,23ZM26,32C28.76,32 31,29.76 31,27C31,24.24 28.76,22 26,22C23.24,22 21,24.24 21,27C21,29.76 23.24,32 26,32ZM22.8,27C22.8,28.77 24.23,30.2 26,30.2C27.77,30.2 29.2,28.77 29.2,27C29.2,25.23 27.77,23.8 26,23.8C24.23,23.8 22.8,25.23 22.8,27Z"
-            android:strokeColor="#00000000" android:strokeWidth="1"/>
-    </group>
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+  android:width="150dp"
+  android:height="150dp"
+  android:autoMirrored="true"
```

Comment:
This should not be reversed. Only icons which represent direction should be reversed.

Reply:
Done.